axelrod/tests/strategies/test_memorytwo.py

```diff
@@ -22,22 +22,27 @@ class TestMEM2(TestPlayer):
     }
 
     def test_strategy(self):
+        self.first_play_test(C)
         # Start with TFT
-        self.responses_test([C])
-        self.responses_test([C], [C], [C])
-        self.responses_test([D], [C], [D])
-        # TFT if mutual cooperation on first two rounds
-        self.responses_test([C], [C, C], [C, C])
-        self.responses_test([D], [C, D], [D, D])
+        actions = [(C, C), (C, C)]
+        self.versus_test(opponent=axelrod.Cooperator(),
+                         expected_actions=actions,
+                         attrs={"play_as": "TFT", "shift_counter": 1,
+                                "alld_counter": 0})
+        actions = [(C, D), (D, D)]
+        self.versus_test(opponent=axelrod.Defector(), expected_actions=actions,
+                         attrs={"play_as": "TFT", "shift_counter": 1,
+                                "alld_counter": 0})
         # TFTT if C, D and D, C
```

Comment:
I've just checked by removing each case and the test fails so this test if hitting both of them. :+1: